boards/arm/b_l072z_lrwan1/b_l072z_lrwan1.dts

```diff
@@ -73,6 +74,26 @@ arduino_i2c: &i2c1 {};
 
 &spi1 {
 	status = "okay";
+	cs-gpios = <&gpioa 15 0>;
```

Comment:
It should be GPIO_ACTIVE_LOW. Those gpio flags for CS are not used now, but maybe at some point they will be used. Let's be prepared :)

Reply:
Ack